docs/migrations/chain-upgrade-guide-043.md

```diff
@@ -108,12 +114,16 @@ Now that our node is initialized and we are ready to start a new `simapp` chain,
 
 ### Cosmovisor Setup
 
-First, install or update `cosmovisor`:
+Install the `cosmovisor` binary:
 
 ```
-go get github.com/cosmos/cosmos-sdk/cosmovisor/cmd/cosmovisor
+go install github.com/cosmos/cosmos-sdk/cosmovisor/cmd/cosmovisor@v0.1.0
 ```
 
+::: tip
+If you are using go `v1.15` or earlier, you will need to change out of the current directory and run `go get` instead of `go install` and then switch back into the `cosmos-sdk` repository.
+:::
```

Comment:
instead of repeating this section, let's add a link to the section above.

Reply:
We are not repeating a section within the same document. I think you are referring to the note within the cosmovisor readme. This tip is also more specific to the instructions with reference to `cosmos-sdk`. I think it would be good to leave it here within the instructions rather than pointing users to a separate document mid-instructions.